vuu-ui/packages/vuu-popups/src/dialog/Dialog.css

```diff
@@ -1,3 +1,12 @@
+@font-face {
```

Comment:
Do we need this @font-face{} here as well? to avoid duplication we could move it somewhere where the styling can be shared with both components. Maybe in vuu-theme/css/global.css?

Reply:
https://github.com/ScottLogic/finos-vuu/pull/37/commits/9dd148b1156af901445235f4d48012460731428a